### PR TITLE
React & Redux group project - Space Travelers' Hub

### PR DESCRIPTION
**Pull request by gdvl777**
## Description

This project is to build a web application that allows users to book rockets and join space missions using real-time data from the SpaceX API. The application comprises three main sections: Rockets, Missions, and My Profile. 

## Rockets
image

## Dragon
image

## Missions
image

##My Profile
image

Also, our live demo:
https://gdvl777.github.io/space_traveler/

### CODE REVIEW

File: src/components/navigation.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            <NavLink to="/space-traveler" className="navlink300" activeclassname="active-link">Rockets</NavLink>
+          </li>
+          <li>
+            <NavLink to="/dragons" className="navlink300" activeclassname="active-link">Dragons</NavLink>
+          </li>
+          <li>
+            <NavLink to="/missions" className="navlink300" activeclassname="active-link">Missions</NavLink>
+          </li>
+          <li>
+            <NavLink to="/myprofile" className="navlink300" activeclassname="active-link">My Profile</NavLink>
```
Comment (Owusu-Desmond) [inline src/components/navigation.jsx:24]: - [ ] To ensure clarity on the current page, please ensure that the link for that page is underlined. To emphasize active navigation links with React Router, utilize the NavLink component. This component generates a link inside your navigation bar and offers various styling options through its props. For further details on implementing the NavLink component, please see this article: [How to Highlight the Active NavLink](https://javascript.plainenglish.io/react-router-how-to-highlight-the-active-navlink-50ecb95384ca#:~:text=In%20React%20Router%2C%20you%20can,the%20activeClassName%20and%20activeStyle%20props). :blush:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @gdvl777, @fmanimashaun, and @EvansSnave  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/navigation.jsx (hunk 1 above)
Comment (fmanimashaun) [reply src/components/navigation.jsx]: an active state has been added to the navigation as requested.

**Review by Bantealem — APPROVED** [review]
# Status: Approved  approved

Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Screenshot from 2023-09-27 20-56-29


## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set-up of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**